generated/php/google-cloud-resourcemanager-v2/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "resourcemanager/resourcemanager",
+  "version": "0.1.0",
+  "author": "Google LLC",
+  "url": "https://github.com/googleapis/googleapis",
+  "description": "Google Cloud Resource Manager API client for PHP",
+  "require": {
+    "google/gax": "^0.25.0",
```

Comment:
This should be 0.27. You will need to generate with image 0.5.4-rc02. If you prefer we can edit this PR manually to update this setting.